hasura/migrations/default/1662391842396_interaction_events/up.sql

```diff
@@ -0,0 +1,37 @@
+CREATE TABLE "public"."interaction_events" (
+  "user_id" integer NULL,
+  "profile_id" integer NULL,
+  "created_at" timestamptz NOT NULL DEFAULT now(),
+  "updated_at" timestamptz NULL DEFAULT now(),
+  "event_type" text NOT NULL,
+  "event_subtype" text NULL,
+  "data" jsonb NULL,
+  "circle_id" integer NULL,
+  "org_id" integer NULL,
+  "id" serial NOT NULL,
+  PRIMARY KEY ("id")
+);
+
+CREATE OR REPLACE FUNCTION "public"."set_current_timestamp_updated_at"
```

Comment:
remove this

Reply:
done

Reply:
@levity why remove this trigger? do we have a trigger covering all tables already?

Reply:
yeah, this is the one that's caused deployment failures in the past because it's not owned by the current psql user